Mathlib/Algebra/Squarefree/Basic.lean

```diff
@@ -48,7 +51,7 @@ theorem squarefree_one [CommMonoid R] : Squarefree (1 : R) :=
 
 @[simp]
 theorem not_squarefree_zero [MonoidWithZero R] [Nontrivial R] : ¬Squarefree (0 : R) := by
-  erw [not_forall]
+  rw [squarefree_def, not_forall]
```

Comment:
Or just 
```suggestion
  rw [Squarefree, not_forall]
```
which avoids needing the new theorem.

Reply:
`Squarefree.eq_def` works too, if you really (for whatever reason) want to be rewriting with a theorem